internal-packages/next-dev/src/index.ts

```diff
@@ -50,7 +50,7 @@ export type DevBindingsOptions = {
 export type Binding =
 	| { type: 'kv'; id: string }
 	| { type: 'r2'; bucketName: string }
-	| { type: 'd1'; databaseName: string }
+	| { type: 'd1'; databaseName: string; databaseId?: string }
```

Comment:
Should we make the `databaseName` field optional?

Reply:
mh.... yeah good point! 😕 

Yes let's! 🙂

Reply:
I wondered about that approach but was worried it would complicate the bindings type too much.
If you are happy with it then that's good for me.

Reply:
To me it feels like a more sensible type less likely to confuse people 🤔 (hopefully), since otherwise it would be less clear if and when users should provide the db name 🤔

I'd go with this since you're ok with it 😄